R/pkg/R/generics.R

```diff
@@ -581,6 +579,10 @@ setGeneric("unionAll", function(x, y) { standardGeneric("unionAll") })
 #' @export
 setGeneric("where", function(x, condition) { standardGeneric("where") })
 
+#' @rdname with
+#' @export
+setGeneric("with")
```

Comment:
it looks like `with` is duplicated:
https://github.com/olarayej/spark/blob/SPARK-11031/R/pkg/R/generics.R